src/components/SubSection/subsection.module.css

```
@@ -0,0 +1,20 @@
+.container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 24px;
+    text-align: center;
+}
+
+.title {
+    color: var(--blue-700);
+    font-size: 36px;
```

Acredito que esse título pode ser um pouquinho menor

```suggestion
    font-size: 32px;
```